notebooks/fill_counties_api.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
@esloch if you want to receive a review for notebooks, you probably want to integrate it with https://www.reviewnb.com/

Reply:
Interesting platform,  I didn't know it. At least describe the main changes in the commit messages. But notebooks are notoriously hard to review. In my opinion, all relevant code should be kept outside them and Notebooks should only be used for demonstrating features.